sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py

```diff
@@ -326,7 +326,7 @@ def get_table_client(self, table_name: str, **kwargs) -> TableClient:
         return TableClient(
             self.url,
             table_name=table_name,
-            credential=self.credential,
+            credential=self.credential,  # type: ignore
```

Comment:
Why do we have to suppress the type here?

Reply:
The AccountHostsMixin expects a sync token credential.